lib/install.js

```diff
@@ -144,6 +164,24 @@ exports.install = function(targets, options) {
     .then(function() {
       return saveInstall();
     })
+    .then(function () {
+      var hooks = config.pjson.hooks;
+      if (hooks) {
+        return new Loader().import(hooks.postinstall)
+          .then(function (m) {
+            if (m.default && typeof m.default === 'function') {
+              ui.log('info', 'Running postinstall hook...');
+              return Promise.resolve(m.default());
+            }
+          })
+          .catch(function (error) {
+            if (error) {
+              ui.log('err', 'Error during postinstall hook');
+              ui.log('err', error);
+            }
+          });
+      }
```

Comment:
Since this code is a direct duplication of the above, it is probably worth generalising it into a function.

Happy to have that function at the top of this `install.js` file for the mean time - it doesn't yet have to be its own file.